google-maps/README.md

```diff
@@ -0,0 +1,16 @@
+# cljsjs/google-maps
+
+[](dependency)
+```clojure
+[cljsjs/google-maps "3.18-0"] ;; latest release
+```
+[](/dependency)
+
+This jar comes with `deps.cljs` as used by the [Foreign Libs][flibs] feature
+of the Clojurescript compiler.
+
+This library provides an extern file required for compilation in advanced
+mode.
```

Comment:
I wonder if this could be made clearer. I.e. people should not expect this thing to do anything, they are not supposed to `: require` it and they'll need an extra script tag to make use of Google Maps JS. I think since most Jars do this stuff it's important here to highlight the differences to avoid confusion. :smile_cat: